src/constants/data/patientsData.js

```diff
@@ -15,13 +14,16 @@ const patientsData = [
     person_authorized: "",
     phone_authorized: "",
     guardianship: false,
+    diagnosis: ["depresja", "nerwica", "nerwica", "nerwica", "nerwica"],
+    medicines: ["witamina c", "ibuprom"],
+    hospitalization_times: 3,
+    code: "A52.46",
```

Comment:
Tutaj pytanie w kontekście przyszłościowym: czy będzie mapowanie w bazie danych między kodem rozpoznania choroby  a diagnozą?

Reply:
Musimy mieć coś takiege. W sumie mogłoby by to być przekazywane w diagnosis jako obiekt:
`diagnosis: [{name: "depresja", code: "A52.46"}}`

Reply:
Pewnie tak, ale bardziej chodziło mi o to że przy dodawaniu nowego pacjenta wpisuje się kod rozpoznania (o ile miał już wcześniej postawioną diagnozę). Nie wiem jak dalej wygląda flow i czy lekarz po postawieniu diagnozy wpisuje jej nazwę czy tylko kod. Jeśli tylko kod to będziemy potrzebować mapowania z kodu rozpoznania ICD-10 na nazwę

Reply:
A to już rozumiem, w takim razie faktycznie trzeba takie mapowanie zrobić przy zapytaniu do bazy tworzącym nowego pacjenta. Jeśli chodzi o diagnozę to wydaje mi się, że lekarz nic nie będzie wpisywać, a jedynie wybierze jedną z opcji (nazw), a kod zostanie zmapowany.